var/spack/repos/builtin/packages/py-pyqt5/package.py

```diff
@@ -46,5 +68,35 @@ def configure_args(self):
                          '--qsci-api-destdir', self.prefix.share.qsci])
         return args
 
+    @when('@5.14:')
+    def build(self, spec, prefix):
+        with working_dir(self.stage.source_path):
+            sip = Executable(join_path(spec['py-sip'].prefix.bin, 'sip-install'))
+            sip(*self.build_args())
+
+    @when('@5.14:')
+    def build_args(self):
+        args = [
+            '--verbose',
+            '--confirm-license',
+            '--qmake', self.spec['qt'].prefix.bin.qmake,
+            '--target-dir', join_path(python_platlib, 'PyQt5'),
+            '--jobs', str(inspect.getmodule(self).make_jobs),
```

Comment:
```suggestion
            '--jobs', str(make_jobs),
```
Inspect is only needed in package base classes. Also unclear if conversion to string is actually needed or not

Reply:
The conversion to string is needed to prevent: `Error: TypeError: expected string or bytes-like object`.